.github/workflows/update_knapsack_report.yml

```diff
@@ -29,14 +29,14 @@ jobs:
 
     env:
       RAILS_ENV: test
+      github_token: ${{ secrets.GITHUB_TOKEN }}
 
     steps:
     - uses: actions/checkout@v2
-    - name: Set up Ruby 3.1
-      uses: actions/setup-ruby@v1
+    - name: Set up Ruby
+      uses: ruby/setup-ruby@v1
       with:
-        ruby-version: 3.1
```

Comment:
why? we want to run the same version of ruby as the app because it's gonna run the tests, right?

Reply:
Because this was throwing a warning "invalid key". Yes we want this, but this detect the app version automatically
![image](https://github.com/rootstrap/rails_api_base/assets/22925144/b62989ff-ca4a-434e-9cdb-2aa6004d275d)